docs/report/bom.md

```diff
@@ -0,0 +1,113 @@
+---
+title: BOM Generation
+---
+
+## BOM Generation
+The bill of materials is an essential part of the documentation that needs to be send to the facctory. A sinple csv export is OK to be importet into SMT machines. But for human redable documentation it might not be suficcient. Additional information ist needed. The Inventree report system allows to generate BOM well formatted BOM reports. 
```

Comment:
Spelling

- "send" - "sent"
- "facctory" - "factory"
- "sinple" - "simple"
- "importet" - "imported"
- "redable" - "readable"
- "suficcient" - "sufficient"
- "ist" - "is"